### DISCUSSION TITLE
How to remove another stories from library component

### DISCUSSION THREAD

**Discussion by d-strygwyr** [storybookjs/storybook#21372] [General]
in my project, I'm using chakra-ui and I don't want to show the chakra-ui stories in my storybook 

how to remove chakra-ui stories section?
![image](https://user-images.githubusercontent.com/49369600/222678285-1ce8523b-5a4e-4a2f-8fd8-503d113be526.png)

**Comment by ccdlt** [comment]
Any update on this? I'm having the same issue. Nowhere in my code am I referencing this Chakra UI storybook and I can't seem to get rid of it.

**Reply by ccdlt** [reply]
Figured out how to remove it. Add this to your `.storybook/main.ts`

```   
refs: {
    "@chakra-ui/react": {
      disable: true,
    },
  },
  ```
